src/backend/lib/local-upgrade-info.js

```diff
@@ -0,0 +1,102 @@
+const path = require("path");
+const fs = require("fs");
+const RWLock = require("rwlock");
+
+const UPGRADE_INFO_FILENAME = require("./constants").UPGRADE_INFO_FILENAME;
```

Comment:
```suggestion
const { UPGRADE_INFO_FILENAME } = require("./constants");
```

Really not bothered about this, but I think most of the code in Mapeo uses destructuring for this kind of import.